### DISCUSSION TITLE
Azure self deployment

### DISCUSSION THREAD

**Discussion by mike12345567** [Budibase/budibase#987] [Ideas]
<p align="center">
<img src="https://user-images.githubusercontent.com/4407001/104211118-94b2ba00-542b-11eb-8501-781f88724371.png"  width="128">
</p>

Budibase now supports self hosting with the help of [Docker](https://www.docker.com/)/[Docker compose](https://docs.docker.com/compose/) but we want to keep making it easier to build your ideas with our tools. As part of this we plan to introduce more self hosting deployment options, one option for this is support for Azure.

If you would be interested in a simple Azure deployment of Budibase this is the place to talk about it! We would love to know your use case for this and why you would prefer this over any other deployment option.

**Comment by samthompsonkennedy** [comment]
Hi Team,
I'm looking at deploying this ASAP on Azure, is there any support for this yet? Or do you guys have any pointers?

I've got a bit of pressure to hit this up and running eek!
The company runs everything internally on Azure, their requirement for any news apps are that they are hosted there.

**Reply by shogunpurple** [reply]
@samfroggy we do not have a direct azure deployment but you should be able to run an azure Linux VM and set it up like any other machine:

https://azure.microsoft.com/en-gb/pricing/details/virtual-machines/linux/

- Run your VM on azure 
- SSH into it
- Follow the instructions at https://docs.budibase.com/getting-started
